tests/test_router_bgp.rb

```diff
@@ -399,12 +400,38 @@ def test_default_cluster_id
   def test_enforce_first_as
     asnum = 55
     bgp = RouterBgp.new(asnum)
-    bgp.enforce_first_as = true
-    assert(bgp.enforce_first_as,
-           'bgp enforce-first-as should be enabled')
-    bgp.enforce_first_as = false
-    refute(bgp.enforce_first_as,
-           'bgp enforce-first-as should be disabled')
+    if platform == :ios_xr
+      @device.cmd('term width 300')
```

Comment:
I put it in because I was seeing munged output in my show command. But I was also developing the regexp. Anyway - rerun the test without this addition and they are fine. So will remove.